src/renderer/components/recording-page/recording-page.test.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable no-console */
+/* eslint-disable jest/prefer-expect-assertions */
+
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+
+import { noOp } from '../../env-and-consts';
+import mediaService from '../../services/media';
+
+import { RecordingPage } from './index';
+
+jest.mock('../../services/media', () => ({
+  switchOnAudioInput: jest.fn(noOp),
+  stopRecording: jest.fn(noOp),
+  startRecording: jest.fn(noOp),
+  audioBlobToWavArrayBuffer: jest.fn(noOp),
+  createNewAudioGraph: jest.fn(noOp),
+  createOscillator: jest.fn(noOp),
+  playAudioNode: jest.fn(noOp),
+}));
+
+const originalError = console.error;
```

Comment:
Let's not suppress these warnings - they don't break the tests and it might be worthwhile to address those in the future.